.travis.yml

```diff
@@ -13,9 +13,10 @@ cache:
     - $HOME/.gradle/wrapper/
 addons:
   sonarcloud:
+    organization: 'ioet'
 after_success:
  - bash <(curl -s https://codecov.io/bash)
- - ./gradlew sonarqube -Dsonar.projectKey=$PROJECT_KEY -Dsonar.organization=$ORGANIZATION -Dsonar.host.url=$SONAR_HOST_URL -Dsonar.login=$SONAR_GITHUB_TOKEN
+ - sonar-scanner
```

Comment:
Let me see if I understood it correctly:
sonar-scanner is being used by the CI
gradlew sonarqube will run it locally

is it right? @nishedcob

Reply:
Yes, yesterday I got this working using the command recommended for running this locally (which is sufficient enough to get Travis to run static analysis on a commit by commit level, but we should have seen SonarCloud comment on this PR as part of the integration, so this morning I made this step `sonar-scanner` once more to use the Travis plugin that supposedly has everything work out of the box. I think it is unfortunately too late to see if the PR integration will work with this PR (I imagine that the hook activates on PR creation), so we will have to keep an eye out for this in future PRs and/or see what else needs to be fixed.